<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Endereco</class>
 <widget class="QDialog" name="Endereco">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Endereço</string>
  </property>
  <widget class="QPushButton" name="pushButton_procurar">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>480</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Procurar</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_clientes">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>531</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Entre com nome, telefone ou celular</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_endcadast">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>621</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_apagar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>830</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Apagar Endereço</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cidade">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>770</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[Cidade]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nomecliente">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[nome]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancelar">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>830</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_enderecos">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>531</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gravar">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>830</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Gravar Endereço</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Cidade</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_procurarcliente">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Procurar Cliente</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_cidade">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>450</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_estados">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_procurarua">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Entre com bairro ou rua&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Cliente:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cep">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>770</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[CEP]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_procurarcliente">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_bairro">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>770</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[Bairro]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_estado">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>770</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[Estado]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rua">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>740</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Endereços Cadastrados</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Endereço de Cliente</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_numero">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>740</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_numero">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>720</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Número</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_numapto">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>740</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>720</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Apto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_novocliente">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Novo
Cliente</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_bloco">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>740</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>720</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Bloco</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
